packages/grid/x-data-grid-pro/src/hooks/features/rowReorder/useGridRowReorder.tsx

```diff
@@ -81,6 +81,7 @@ export const useGridRowReorder = (
       });
 
       originRowIndex.current = apiRef.current.getRowIndexRelativeToVisibleRows(params.id);
+      apiRef.current.setCellFocus(params.id, '__reorder__');
```

Comment:
Do we have a constant for this? I like to avoid magic strings if possible.

Also, why is this change necessary?

Reply:
> Also, why is this change necessary?

It's fixes https://github.com/mui/mui-x/issues/6165. when you drag rows, you are really dragging the first draggable cell and it will not update the `focusedCell` state when you drag. so if you manage the cell to be out of the viewport and the cell is unmounted. it prevents the rows from reordering.